test_helpers/tcs/tcs.go

```diff
@@ -0,0 +1,98 @@
+package tcs
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/tarantool/go-tarantool/v2"
+	"github.com/tarantool/go-tarantool/v2/test_helpers"
+)
+
+type Tcs struct {
+	// conn Connection interface for interacting with Tarantool.
+	doer tarantool.Doer
+}
+
+// GetResponse is a response of a get request.
+type GetResponse struct {
+	Data []struct {
+		Path        string
+		Value       string
+		ModRevision int64 `mapstructure:"mod_revision"`
+	}
+}
+
+// StartTcs starts a Tarantool centralized configuration storage.
+// Returns a Tcs instance and a cleanup function.
+func StartTcs(opts test_helpers.StartOpts) (Tcs, func()) {
```

Comment:
```suggestion
// StartTcs starts a Tarantool centralized configuration storage.
// Returns a Tcs instance and a cleanup function.
func StartTcs(opts test_helpers.StartOpts) *Tcs, func() {
```

I would prefer a separate StopTCS() call here:

```Go
func TestFoo(t *testing.T) {
    helpers.StopTCS(helpers.StartTCS(t, opts))
```

vs

```Go
func TestFoo(t *testing.T) {
    _, cancel := helpers.StartTCS(t, opts)
    defer cancel()

```

Reply:
Refactored